front/src/modules/powerRestriction/helpers/createPathStep.ts

```diff
@@ -127,8 +124,7 @@ export const createCutAtPathStep = (
     id: nextId(),
     positionOnPath: cutAtPosition,
     coordinates: coordinatesAtCut,
-    track: trackSectionRangeAtCut.track_section,
-    offset: offsetAtCut,
+    ...trackOffset,
```

Comment:
No need for `offset: mmToM(trackOffset.offset)` here? Seems like the inputs are the same as in `createPathStep()`, and the result is passed to `cutPowerRestrictionRangesV2()` which may insert the whole thing incl. `offset` into the store?

Reply:
Yes you're right, I thought it was a different case, but after a check it was not ! Thanks :pray: